### DISCUSSION TITLE
how to disable right prompt `PROMPT_COMMAND_RIGHT `??

### DISCUSSION THREAD

**Discussion by yuuyins** [nushell/nushell#8040] [Q&A] [answered]
i tried null it, removing it and its function... it is still there, displaying datetime. how to actually disable it??

**Comment by fdncred** [comment, marked as answer]
Setting it to an empty string effectively disables it. `let-env PROMPT_COMMAND_RIGHT = ""`

**Reply by yuuyins** [reply]
thank you!
